components/card/card.js

```diff
@@ -1 +1,27 @@
-export function createCharacterCard() {}
+export function createCharacterCard(
+  characterImage,
+  characterName,
+  characterStatus,
+  characterType,
+  characterOccurances
+) {
+  const card = document.createElement("li");
```

Comment:
Maybe you need to add the .classList to your card element, if you delete the "li" from line 9. (next comment)